cylc/flow/cfgspec/workflow.py

```diff
@@ -1299,14 +1311,31 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                 job scripts to run.
 
             ''') + REPLACED_BY_PLATFORMS):
-                Conf('batch system', VDR.V_STRING)
-                Conf('batch submit command template', VDR.V_STRING)
+                Conf('batch system', VDR.V_STRING, desc='''
+                    Batch/Queuing system to submit task jobs to.
+
+                    .. seealso::
+
+                       Kept for back compatibility but replaced by
+                       :cylc:conf:`global.cylc[platforms][<platform name>]
+                       job runner`.
+                ''')
+                Conf('batch submit command template', VDR.V_STRING, desc='''
+                    Override the default job submission command for the chosen
+                    batch system.
+
+                    .. seealso::
+
+                       Kept for back compatibility but replaced by
+                       :cylc:conf:`global.cylc[platforms][<platform name>]
+                       job runner command template`.
```

Comment:
Ditto.

Reply:
Again, not straightforward because there's a rename too.